init.sh

```diff
@@ -3,7 +3,7 @@ TOOLSPATH=/cvmfs/fcc.cern.ch/sw/0.8.3/tools/
 OS=`python $TOOLSPATH/hsf_get_platform.py --get os`
 
 # source FCC externals
-source /cvmfs/fcc.cern.ch/sw/views/releases/externals/96b.0.0/x86_64-$OS-gcc8-opt/setup.sh
+source /cvmfs/fcc.cern.ch/sw/views/releases/externals/94.2.0/x86_64-$OS-gcc62-opt/setup.sh
```

Comment:
I guess we can bear this for the moment, but we should be more flexible about dependencies and not hard code paths